src/smt/model_blocker.cpp

```diff
@@ -24,7 +24,8 @@ namespace CVC4 {
 
 Expr ModelBlocker::getModelBlocker(const std::vector<Expr>& assertions,
                                    theory::TheoryModel* m,
-                                   BlockModelsMode mode)
+                                   BlockModelsMode mode,
+                                   const std::vector<Node>* nodesToBlock)
```

Comment:
Can you make this const std::vector< Node >& ?  This is the more common way of passing a vector.